pages/express-relay/websocket-api-reference.mdx

```diff
@@ -256,17 +257,17 @@ Refer to the examples below, one for each of the status options in EVM and SVM:
 	<Tabs.Tab>
 	```json
 	// lost
-	/// The bid lost the auction.
-    /// The result will be None if the auction was concluded off-chain and no auction was submitted to the chain.
-    /// The result will be not None if another bid were selected for submission to the chain.
-    /// The signature of the transaction for the submitted bid is the result value.
+    // The bid lost the auction.
+    // The result will be None if the auction had no winner (because all bids were found to be invalid).
```

Comment:
is it more accurate to say the bid_status json will have a `result` field if the auction has a winner and otherwise will not have a `result` field?

Reply:
yep updated